### ISSUE TITLE
XF86* key support

### ISSUE THREAD

**Issue by TLATER** [ch11ng/exwm#55]
Using bindings such as:

```elisp
  (exwm-input-set-key (kbd "s-<f8>") (lambda ()
                                       (interactive)
                                       (volume-change 5)))
  (exwm-input-set-key (kbd "<XF86AudioLowerVolume>") (lambda ()
                                                       (interactive)
                                                       (volume-change -5)))
```

Does work to bind functions to keys as long as they are used in a normal emacs buffer. However, when in an X window-buffer the keys don't seem to be caught by emacs.

**Comment by ch11ng** [comment]
Special keys like <kbd>\<XF86AudioLowerVolume\></kbd> are currently not recognized. So with your code EXWM shall not even be able to start. But keys like <kbd>s-\<f8\></kbd> should work. Perhaps you evaluated this code after EXWM has started?

**Comment by TLATER** [comment]
Yup, that solved it... I used the eval commands to execute these, so I suppose that's how it managed to bind the special keys regardless. Strange that other keys work, though.

**Comment by ch11ng** [comment]
Fixed in ch11ng/xelb@ae7da13819420dbebf17b2635cc43c38cf269627.
